spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -215,6 +215,84 @@ definitions:
             fill: SubSystemReport
             desc: Reported status of individual subsystems
 
+ - StatusJournalItem:
+    short_desc: Sub-system Status report
+    embedded_type: true
+    desc: >
+      Report the general and specific state of a sub-system.  If the generic
+      state is reported as initializing, the specific state should be ignored.
+    fields:
+        - uptime:
+            type: u32
+            desc: Milliseconds since system startup
+        - component:
+            type: u16
+            desc: Identity of reporting subsystem
+            fields:
+              - 0-15:
+                  desc: Subsystem
+                  values:
+                      - 0: Primary GNSS Antenna
+                      - 1: Measurement Engine
+                      - 2: Corrections Client
+                      - 3: Differential GNSS Engine
+                      - 4: CAN
+                      - 5: Wheel Odometry
+                      - 6: Sensor Fusion Engine
+        - generic:
+            type: u8
+            desc: Generic form status report
+            fields:
+              - 0-7:
+                  desc: Generic
+                  values:
+                      - 0: OK/Nominal
+                      - 1: Initializing
+                      - 2: Unknown
+                      - 3: Degraded
+                      - 4: Unusable
+        - specific:
+            type: u8
+            desc: Subsystem specific status code
+
+ - MSG_STATUS_JOURNAL:
+    id: 0xFFFD
+    short_desc: Status report journal
+    desc: >
+        The status journal message contains past status reports (see MSG_STATUS_REPORT)
+        and functions as a error/event storage for telemetry purposes.
+    fields:
+        - reporting_system:
+            type: u16
+            desc: Identity of reporting system
+            fields:
+              - 0-15:
+                 desc: System
+                 values:
+                     - 0: Starling
+                     - 1: Precision GNSS Module (PGM)
+        - sbp_version:
+            type: u16
+            desc: SBP protocol version
+            fields:
+              - 8-15:
+                  desc: SBP major protocol version number
+              - 0-7:
+                  desc: SBP minor protocol version number
+        - n_status_reports:
+            type: u32
+            desc: Total number of status reports sent since system startup
+        - packet_index:
+            type: u8
+            desc: Index of this packet in the status journal
+        - n_packets:
```

Comment:
from discussion this is meant to represent a multi-message sequence

Reply:
We haven't used the terminology "packet" yet in SBP fields yet, sequence is much more common... for consistency we should switch to using sequence instead of packet.